c++/measures/g4_iw.cpp

```diff
@@ -52,16 +53,18 @@ namespace cthyb {
 
     // Allocate temporary NFFT two-frequency matrix M
     {
-      int resize_factor = 3; // How much bigger should the large mesh bee???
-      int nfreq         = resize_factor * std::max(n_fermionic, n_bosonic - 1 + n_fermionic);
+      // Determine maximum Matsubara index needed
+      int nfreq = std::max(3 * n_fermionic, n_bosonic + n_fermionic);
       gf_mesh<imfreq> iw_mesh_large{beta, Fermion, nfreq};
+      // In case of fermionic measurement, large mesh only needed for first component
```

Comment:
That's awkward. Do you get out of bounds access? (valgrind)

Deciding on the fftw size is not directly possible from the nfft_init,
you have to rather use the nfft_init_guru as in
https://bitbucket.org/Wentzell/ctint/src/d2b93f0302a8b6ed97def5879abd71e9a7b691c4/c++/nfft_buf.hpp?fileviewer=file-view-default

But why do you expect this to be linked to the oversampling factor?